### PR TITLE
VATEAM-90582: Add 21-4140 Digital Form to seed script

### PR DESCRIPTION
**Pull request by derekhouck**
## Description
Adds VA Form 21-4140 to the Digital Form seed script created in #19025.

- Closes department-of-veterans-affairs/va.gov-team#90582

## Testing done
Ran the script in my local environment and verified the creation of a Digital Form with the correct info for VA Form 21-4140.

## Screenshots
VA Form 21-4140 as generated by the script:
Screenshot 2024-08-26 at 3 50 06 PM

## QA steps

As an administrator
1. Open the command line and navigate to the `va.gov-cms` repo.
2. Start the CMS: `ddev start`
3. Enter the terminal for the web server: `ddev ssh`
4. Run the script: `drush scr scripts/content/digital-forms.php`
5. Go to Manage > Content
   - [ ] Verify that a Digital Form was created with the title "Employment Questionnaire"
6. Click the title
7. Validate the following fields
   - [ ] Form number: 21-4140
   - [ ] OMB Number: 2900-0079
   - [ ] Steps: 1 step titled "Veteran's personal information"

### Definition of Done

- [x] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [x] Automated tests have passed.
- [x] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [x] No announcement is needed for this code change.
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

File: scripts/content/digital-forms.php [hunk 1]
```diff
@@ -21,48 +21,76 @@ function run() {
...
-  string $title = 'Script Generated Step',
-  bool $includeDob = TRUE,
+  array $values = [],
 ): Paragraph {
   return Paragraph::create([
     'type' => 'digital_form_name_and_date_of_bi',
-    'field_title' => $title,
-    'field_include_date_of_birth' => $includeDob,
+    'field_title' => $values['title'] ?? 'Script Generated Step',
+    'field_include_date_of_birth' => $values['includeDob'] ?? TRUE,
```
Comment (ryguyk) [inline scripts/content/digital-forms.php:94]: The camel-cased `includeDob` feels inconsistent. Perhaps this should be in snake case?

**Review by ryguyk — CHANGES_REQUESTED** [review]
A very small, somewhat pedantic change request, but I think it will be good for code cleanliness.

File: scripts/content/digital-forms.php (hunk 1 above)
Comment (derekhouck) [reply scripts/content/digital-forms.php]: Good call. Updated. Switching too much between Javascript and Drupal.

**Review by ryguyk — APPROVED** [review]
Approved. Waiting to get integration branches squared away before merging.

**Pull request merged**